src/app/modules/main/wallet_section/send/network_route_model.nim

```diff
@@ -198,119 +163,96 @@ QtObject:
       self.items[i].amountOut = ""
       self.dataChanged(index, index, @[ModelRole.AmountIn.int, ModelRole.ToNetworks.int, ModelRole.HasGas.int, ModelRole.Locked.int, ModelRole.AmountOut.int])
 
-  proc getRouteDisabledNetworkChainIds*(self: NetworkModel): seq[int] =
+  proc getRouteDisabledNetworkChainIds*(self: NetworkRouteModel): seq[int] =
     var disbaledChains: seq[int] = @[]
     for item in self.items:
-      if not item.getIsEnabled():
+      if not item.getIsRouteEnabled():
         disbaledChains.add(item.getChainId())
     return disbaledChains
 
-  proc getRouteLockedChainIds*(self: NetworkModel): string =
+  proc getRouteLockedChainIds*(self: NetworkRouteModel): string =
     var jsonObject = newJObject()
     for item in self.items:
       if item.getLocked():
         jsonObject[$item.getChainId()] = %* ("0x" & item.getLockedAmount())
     return $jsonObject
 
-  proc updateRoutePreferredChains*(self: NetworkModel, chainIds: string) =
+  proc updateRoutePreferredChains*(self: NetworkRouteModel, chainIds: string) =
     try:
       for i in 0 ..< self.items.len:
         let index = self.createIndex(i, 0, nil)
         defer: index.delete
-        self.items[i].isPreferred = false
-        self.items[i].isEnabled = false
+        self.items[i].isRoutePreferred = false
+        self.items[i].isRouteEnabled = false
         if chainIds.len == 0:
-          if self.items[i].getLayer() == 1:
-            self.items[i].isPreferred = true
-            self.items[i].isEnabled = true
+          let networkItem = self.delegate.getNetworkItem(self.items[i].getChainId())
```

Comment:
I see this call is here, just to figure out the layer of a network with a certain id, why don't we keep then the layer prop in `NetworkRouteItem` as well, cause that's something that doesn't change (I mean certain chain id is always tied for a certain layer)?